src/content/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Upgrade to the new synthetic monitor runtime to prevent impacts to your synthetic monitors'
+summary: 'Support for legacy synthetics runtimes and Containerized Private Minion will be concluded on October 9, 2024'
+releaseDate: '2024-02-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui'
+getStartedLink: ''
+---
+
+On October 9, 2024, New Relic will end-of-life (EOL) legacy synthetics runtimes including our legacy Chrome 72 (and older) and Node 10 (and older) Synthetics runtimes and the Containerized Private Minion (CPM). Please refer to our [transition guide](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime/) to update to the latest runtimes or contact support. 
```

Comment:
I know this might look unusual and like a ton of bullets, but the sentence felt overburdened to include these two separate items. You might have a better suggestion tho -- maybe we can split them into their own sentences. What do you think?

Reply:
Looks good to me, thanks!